keps/sig-node/4112-passdown-resources-to-cri/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Pass down resources to CRI
+kep-number: 4112
+authors:
+  - "@marquiz"
+  - "@askervin"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
```

Comment:
should be implementable unless you plan to merge before it's complete

Reply:
Thanks, changed.